functionalTest/test/specs/example.e2e.js

```diff
@@ -0,0 +1,81 @@
+const dashboard = require('../pageobjects/dashboard.page');
+const reusableObj = require('../utilities/reusableLibrary');
```

Comment:
reusableLibrary doesn't exist - its causing the test run to fail

Reply:
Removed reusableLibrary import from spec file. But added one reusableLibrary file in utility folder for future purposes. So that whenever we can write some reusable methods in it.